backend-models/service-tests/sectionedservice-test.txt

```diff
@@ -0,0 +1,551 @@
+sectionedservice.php
+
+----------------------------------
+getGroupsTest() line 100:
+----------------------------------
+
+*********
+ Pre-req
+*********
+
+(checklogin) IS TRUE
+
+//// PRE-REQ Values /////
+
+send
+{
+$opt = "GRP"
+$courseid = "1"
+$coursevers = "45656"
+}
+
+/////////////////////////
+
+
+*****************
+ Compare results
+*****************
+
+If $debug is empty then successfully gathered all groups
+
+*****************
+ Service output
+*****************
+
+This service gather groups and if the groups change (unclear in code)
+this will affect:
+
+"groups",
+"grpmembershp",
+"grplst"
+
+----------------------------------
+deleteTest() line 181:
+----------------------------------
+
+*********
+ Pre-req
+*********
+
+(checklogin) IS TRUE
+AND $ha || $studentTeacher IS TRUE
+
+//// PRE-REQ Values /////
+
+send
+{
+$opt = "DEL"
+$sectid != null
```

Comment:
it is not enough to just say that the variable should not have the value NULL.
an actual value needs to be specified.